pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -248,10 +246,15 @@ func (res *{{toTypeName $name}}) FromStackItem(item stackitem.Item) error {
 		return fmt.Errorf("field {{.Field}}: %w", err)
 	}
 {{end}}
-{{end}}
+{{- end}}
 	return nil
 }
-{{end}}`
+{{end -}}`
+
+	srcTmpl = safemethodDefinition +
+		methodDefinition +
+		bindingDefinition
```

Comment:
It doesn't matter, because `methodDefinition` and `safemethodDefinition` are just templates, they won't be added into the resulting code.

But let's have the binding firstly, fixed.